e2e/progressbar/progressbar.e2e-spec.ts

```diff
@@ -0,0 +1,36 @@
+import {ProgressbarPO} from '@agnos-ui/page-objects';
+import {expect, getTest} from '../fixture';
+import {ProgressbarDemoPO} from 'e2e/demo-po/progressbar.po';
+
+const test = getTest();
+test.describe(`Progressbar tests`, () => {
```

Comment:
Instead of testing the attributes individually, could you use a `getSate`  ? (see the `pagination.e2e-spec` example). The same idea than in the UT, we'll test more cases as all side effects of an action will be consider.